.firebaserc

```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": "km-chat-widget"
+    "default": "km-dev-434106"
```

Comment:
@GhiyasKM project name should be dynamic, should not be hardcoded.

Reply:
could you please review now